src/sentry/grouping/result.py

```diff
@@ -1,25 +0,0 @@
-from dataclasses import dataclass, field
-from typing import Optional
-
-from sentry.db.models import NodeData
-from sentry.grouping.variants import BaseVariant
-
-
-@dataclass(frozen=True)
-class CalculatedHashes:
-    hashes: list[str]
-    # `variants` will never be `None` when the `CalculatedHashes` instance is created as part of
-    # event grouping, but it has to be typed including `None` because we use the `CalculatedHashes`
-    # container in other places where we don't have the variants data
-    variants: dict[str, BaseVariant] = field(default_factory=dict)
-
-    def write_to_event(self, event_data: NodeData) -> None:
```

Comment:
Only used in a single location in the codebase and I changed it.
![image](https://github.com/user-attachments/assets/70d3c52f-255e-499b-96f9-bb4b8f8e5f9c)